Fst2List.cpp

```diff
@@ -2361,13 +2803,29 @@ int main_Fst2List(int argc, char* const argv[]) {
         case 's': aa.prMode = PR_SEPARATION; break;
         case 'a': aa.prMode = PR_TOGETHER; break;
         default:
-          error("Invalid arguments: rerun with --help\n");
+          error("Invalid arguments in -f: rerun with --help\n");
           return USAGE_ERROR_CODE;
```

Comment:
Invalid arguments are not only related to the `-f` argument.

Reply:
I changed with your suggestion, thanks